webboot.go

```diff
@@ -8,12 +8,10 @@ import (
 )
 
 var (
-	debug    = func(string, ...interface{}) {}
-	verbose  = flag.Bool("v", true, "verbose debugging output")
-	commands = [][]string{
-		{"date"},
-		{"go", "run", "github.com/u-root/u-root/."},
-	}
+	debug   = func(string, ...interface{}) {}
+	verbose = flag.Bool("v", true, "verbose debugging output")
+	uroot   = flag.String("u", "", "options for u-root")
+	cmds    = flag.String("c", "github.com/u-root/u-root/cmds/core/*", "u-root commands to build into the image")
```

Comment:
What about using the core template rather than cmds/core/* I'm fine either way

Reply:
Also once Louis' stuff gets merged, we should also auto include webboot's commands.

Reply:
yeah, there's a problem doing that but I forget what it is, let me try it.

Reply:
hmm seems to work now ... ok done.

Reply:
next time out we'll add github.com/u-root/weboots/cmds/*